wemake_python_styleguide/violations/consistency.py

```diff
@@ -2107,3 +2109,29 @@ class IterableUnpackingViolation(ASTViolation):
 
     error_template = 'Found an unnecessary iterable unpacking'
     code = 356
+
+
+@final
+class RawStringNotNeededViolation(TokenizeViolation):
+    r"""
+    Forbid the use of raw strings when there is no backslash in the string.
+
+    Reasoning:
+        Raw string are only needed when dealing with \ in the string.
+
+    Solution:
+        Do not prefix the string with r. Use a normal string instead.
+
+    Example::
+        # Correct:
+        s = r'This is a correct use \'
```

Comment:
`s = ` part can be removed

Reply:
See c6825cb4e8c8f27480c5fe98bba4ebbea0546b16